### ISSUE TITLE
Angular 10 : Build error can't find stylesheet to import

### ISSUE THREAD

**Issue by aviralgoyal** [primefaces/primeflex#28]
Just migrated to angular 10 and got this error on build:

```
ERROR in ./src/styles.scss (./node_modules/css-loader/dist/cjs.js??ref--14-1!./node_modules/postcss-loader/src??embedded!./node_modules/resolve-url-loader??ref--14-3!./node_modules/sass-loader/dist/cjs.js??ref--14-4!./src/styles.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Can't find stylesheet to import.
  ╷
1 │ @import './src/_variables';
  │         ^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/primeflex/primeflex.scss 1:9  @import
  /Users/aviral/projects/texma-material/texmamaterialclient/src/styles.scss 6:9                                  root stylesheet
```

**Comment by cagataycivici** [comment]
Fixed via #25

**Comment by Pryanishnik** [comment]
Have similar issue now
```
ERROR in ./node_modules/primeflex/src/_formlayout.scss (./node_modules/css-loader/dist/cjs.js??ref--13-1!./node_modules/postcss-loader/src??embedded!./node_modules/resolve-url-loader??ref--13-3!./node_modules/sass-loader/dist/cjs.js??ref--13-4!./node_modules/primeflex/src/_formlayout.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Undefined variable.
  ╷
2 │     margin-bottom: $fieldMargin;
  │                    ^^^^^^^^^^^^
  ╵
  node_modules\primeflex\src\_formlayout.scss 2:20  root stylesheet
```
